network/transparent_vlan_endpointclient_linux.go

```diff
@@ -171,7 +232,7 @@ func (client *TransparentVlanEndpointClient) PopulateVM(epInfo *EndpointInfo) er
 		// We assume the only possible error is that the namespace doesn't exist
 		logger.Info("No existing NS detected. Creating the vnet namespace and switching to it")
```

Comment:
can we log error here?

Reply:
yes I will add the `existingErr` message to this `logger.Info` statement.